src/components/application_manager/include/application_manager/commands/hmi/response_from_hmi.h

```diff
@@ -61,6 +61,14 @@ class ResponseFromHMI : public CommandImpl {
   void CreateHMIRequest(const hmi_apis::FunctionID::eType& function_id,
                         const smart_objects::SmartObject& msg_params) const;
 
+  /**
+   * @brief Extracts message param value from smart object
+   *
+   * @param message smart object with incoming message
+   * @return value of message param if exists otherwise returns empty string
+   */
+  std::string GetMessageParamValue(const MessageSharedPtr& message) const;
```

Comment:
@LuxoftAKutsan updated description, renamed param and added const in https://github.com/smartdevicelink/sdl_core/pull/1639/commits/602a25a037f158297b4aaf06c36404365b32f9a0